config.yaml

```diff
@@ -2,20 +2,32 @@ options:
   core-interface:
     type: string
     description: Host interface to use for the Core Network.
+  core-interface-mtu:
+    type: int
+    default: 1500
```

Comment:
Do we need set it here as default ?  The default value is determined by Multus maybe we can just mention it in the description.

Reply:
To my earlier comment on the MTU in the UPF: I don't think validation is helpful here.  We really cannot know what equipment is being used, so I thought I'd float this change to see what people think of not having validation other than it needs to be an integer.